content/hi/docs/concepts/architecture/controller.md

```diff
@@ -0,0 +1,167 @@
+---
+title: नियंत्रकों
+content_type: concept
+weight: 30
+---
+
+<!-- overview -->
+
+रोबोटिक्स और ऑटोमेशन में, एक _कंट्रोल लूप_ होता है
+एक नॉन-टर्मिनेटिंग लूप जो एक सिस्टम की स्थिति को नियंत्रित करता है।
+
+यहां कंट्रोल लूप का एक उदाहरण दिया गया है: कमरे में थर्मोस्टेट।
+
+जब आप तापमान सेट करते हैं, तो वह थर्मोस्टेट को बताता है
+आपकी *वांछित अवस्था* के बारे में। वास्तविक कमरे का तापमान है
+*वर्तमान स्थिति*। थर्मोस्टेट वर्तमान स्थिति लाने के लिए कार्य करता है
+उपकरण को चालू या बंद करके वांछित स्थिति के करीब।
+
+{{< glossary_definition term_id="कंट्रोल" length="short">}}
```

Comment:
@akash-kumar-saw For `glossary_definition` shortcode to work as expected we should localize the glossary entry for "controller" (i.e. [en/docs/reference/glossary/controller.md](https://github.com/kubernetes/website/blob/main/content/en/docs/reference/glossary/controller.md)) and incorporate it into this pull request by adding to list of Hindi glossary items inside [/hi/docs/reference/glossary](https://github.com/kubernetes/website/tree/main/content/hi/docs/reference/glossary). Also, please ensure that `term_id` is in English.
```suggestion
{{< glossary_definition term_id="controller" length="short">}}
```